### DISCUSSION TITLE
gui/sp in the tutorial

### DISCUSSION THREAD

**Discussion by kunzaatko** [rktjmp/lush.nvim#15] [Q&A] [answered]
so there is an example of 

```lua
<HighlightGroupName> { bg = <hsl>, fg = <hsl>, gui/sp= <string> }
```
As I understand it `sp` would stand for the `guisp` in help `highlight-guisp`. Is that right? Doesn't it make more sense to make it `guisp` instead?

**Comment by kunzaatko** [comment]
BTW I tried to use `sp` as a drop in substitute for `gui`, but it does not work, so this is not just an alias... Am I right?

**Comment by rktjmp** [comment, marked as answer]
> As I understand it sp would stand for the guisp in help highlight-guisp. Is that right? Doesn't it make more sense to make it guisp instead?

Technically `fg` and `bg` get translated to `guifg` and  `guibg`, so Lush keys are smudged for better DX. You could maybe argue that `gui` should really be `text` or `font`.

- fg => foreground colours
- bg => background colours
- sp => special colours
- gui => ... ~~text~~ graphical user interface effects... :facepalm: 

Improved the docs in d54baed0a8eba78bd9b22671f3141451009ed548.
